app/_data/docs_nav_kuma_dev.yml

```diff
@@ -269,6 +269,10 @@ items:
         url: /policies/applying-policies/
       - text: Understanding TargetRef policies
         url: "/policies/targetref"
+      - text: MeshService
```

Comment:
Should it be in policies? It's a resource

Reply:
Should we just rename the heading? Where else should it go? @lahabana WDYT?

Reply:
Maybe somewhere under `using kuma/Service Discovery & networking`or `Route & traffic shaping`

Reply:
This is meant more as the reference whereas AFAIU those pages are more guides.